Tests/PSSwaggerScenario.Tests.ps1

```diff
@@ -512,7 +512,7 @@ Describe "Composite Swagger Tests" -Tag @('Composite','ScenarioTest') {
 
             Import-Module $PsSwaggerPath -Force
             if((Get-Variable -Name PSEdition -ErrorAction Ignore) -and ('Core' -eq $PSEdition)) {
-                & "powershell.exe" -command "& {`$env:PSModulePath=`$env:PSModulePath_Backup;
+                & "$global:fullPowerShellPath\powershell.exe" -command "& {`$env:Path = '$global:fullPowerShellPath;$env:Path';`$env:PSModulePath=`$env:PSModulePath_Backup;
```

Comment:
>$global:fullPowerShellPath\powershell.exe [](start = 19, length = 41)

Consider using "$env:SystemRoot\System32\WindowsPowerShell\v1.0\PowerShell.exe".

Reply:
Please use this path, I would like to reduce the dependency on run-tests.ps1 and use Invoke-Pester directly on PSCore for this file.
If not, I would like to know the reasons/thoughts.

---
In reply to: [137919841](https://github.com/PowerShell/PSSwagger/pull/324#discussion_r137919841) [](ancestors = 137919841)